utils/emoji_utils.jsx

```diff
@@ -1,16 +1,50 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+const defaultRule = (aName, bName, emojiA, emojiB) => {
```

Comment:
Maybe be a good idea to add some validation for these arguments? At least to make sure the emoji objects exist before trying to access the `.category` properties?